data-raw/data-raw.R

```diff
@@ -0,0 +1,44 @@
+rmfd_guide_shortcuts <- tibble::tribble(
+  ~shortcut, ~name,
+  "pval", "parameter_value_file",
+  "nam", "name_file"
+)
+rmfd_supported_codes <- c("MODFLOW-2005", "MODFLOW-OWHM", "MODFLOW-NWT",
+                          "MODFLOW-LGR", "MODFLOW-CFP")
+rmfd_supported_packages <- tibble::tribble(
+  ~ftype,   ~rmf,
+   'HOB',  'hob',
+  'PVAL',  'pvl',
```

Comment:
I think the `rmf` column should have `pval` instead of `pvl` since the class of a parameter value object has also changed to `pval`. This is used in the top-level functions. Basically `ftype` is the abbreviation used by MODFLOW and `rmf` is the (class) abbreviation used in RMODFLOW.

Reply:
Ah, yes, I did notice this, but then forgot about it later :). Will fix it, then merge to develop.